packages/peregrine/lib/context/cart.js

```diff
@@ -52,6 +55,21 @@ const CartContextProvider = props => {
         derivedCartState
     ]);
 
+    const apolloClient = useApolloClient();
+    const [fetchCartId] = useMutation(CREATE_CART_MUTATION);
+    const fetchCartDetails = useAwaitQuery(CART_DETAILS_QUERY);
+
+    useEffect(() => {
+        // This call initializes the cart if there isn't one. Also, we pass
```

Comment:
I was wonder where is the if check for `cartId`. Aparently it is in the `getCartDetails` function. May you can change the wording of the comments to mention that.

```suggestion
        // cartApi.getCartDetails initializes the cart if there isn't one. Also, we pass
```